### ISSUE TITLE
errors in code on new license/legal code pages

### ISSUE THREAD

**Issue by ericsteuer** [creativecommons/creativecommons.org#835]
i got a private message in slack about a couple of code errors that are showing up on the new legal code pages. the user was viewing in chrome. see attached.
![image](https://user-images.githubusercontent.com/8919762/42647331-89a424da-85b8-11e8-849f-c821f19b0b07.png)

**Comment by little-wow** [comment]
I'm getting different errors in console, but they look like SEO errors to me.

**Comment by robmyers** [comment]
skip-link-focus-fix is in the TwentySixteen WordPress theme and can be accessed OK.

The second one is weird, I don't *think* we should be saving any data blobs. @robinpuga ?

**Comment by altrugon** [comment]
The `skip-link-focus-fix.js` error was due to an extra line of code left behind on the copy/paste procedure from the website theme. I have solved the issue by removing the line below:

`<script type='text/javascript' src='//wp-content/themes/twentysixteen/js/skip-link-focus-fix.js'></script>`

From `creativecommons.org/tools/legalcode-includes/html-head.html` and then run the `update_cc4_includes.py` update script.

For the second error I have to say that I haven't been able to reproduce it on any browser. It's complaining about a missing file, perhaps it's related to the first error, if that is not the case it would help me if the person experiencing this error would click on it and show me the complaining line of code.

The solution for this ticket is on branch `feature/js-errors--dialog-style` remote `https://github.com/creativecommons/creativecommons.org.git`.
